src/gcodeExport.h

```diff
@@ -431,32 +441,32 @@ class GCodeExport : public NoCopy
      * 
      * \param travel_speed The travel speed when priming involves a movement
      */
-    void writePrimeTrain(double travel_speed);
+    void writePrimeTrain(const Velocity& travel_speed);
     
     void writeFanCommand(double speed);
```

Comment:
I did not attempt to touch the fan speeds. Technically it should be a ratio since it's a percentage, but in the case of the fan speeds these are actually stored as a number between 0 and 100 in CuraEngine as well (as opposed to dividing it by 100 upon reading the setting, such as what Flow did). I thought that changing the internal data for all fan speeds was too risky. Currently it still divides this by 100 upon writing the g-code command.

Reply:
I don't think it's risky to store the fan-speed in a floating number instead of an integer. I agree that the change should be made, and so it should be stored as a number between 0 and 1, rather than a number between 0 and 100. That just means that you have to remove a `/ 100` from the code in some location(s).

Don't be afraid for this risk! You can do it!